python_src/src/lamp_py/tableau/jobs/rt_rail.py

```diff
@@ -99,6 +99,8 @@ def __init__(self) -> None:
             "   ve.service_date, vt.route_id, vt.direction_id, vt.vehicle_id, vt.start_time"
             ";"
         )
+        self.ds_batch_size = 1024 * 1024
+        self.db_parquet_path = "/tmp/db_local.parquet"
```

Comment:
moved these because of "too many local variables" from pylint in `update_parquet`